content/docs/command-reference/remote/modify.md

```diff
@@ -298,6 +298,35 @@ For more information on configuring Azure Storage connection strings, visit
 
 <details>
 
+### Click for Dropbox
+
+Please see [Setup a Dropbox DVC Remote](/doc/user-guide/setup-dropbox-remote)
+for a full guide on using Dropbox as DVC remote storage.
+
+- `url` - remote location, in the `dropbox://<dvc-subdirectory>` format:
+
+  ```dvc
+  $ dvc remote modify myremote url dropbox://my-project
+  ```
+
+- `chunk_size` - size of a single data chunk for the files to upload. Needs to
+  be a multiple of 4 MB, with a minimum of 4 MB and maximum of 150 MB. Default
+  `48`.
```

Comment:
Can we add an example like all the other bullets? But first, why would a user want to change this parameter? What's the benefit

Reply:
Added.

Reply:
Thanks @gmrukwa. And can you help me understand what's the benefit of / motivation for being able to change this?

Reply:
Ah never mind I see you added the info. to the option now. Resolving...